ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -169,15 +169,15 @@ open class PopupMenuController: DrawerController {
             )
         )
 
-        separator = Separator()
-        if let separator = separator {
-            separator.backgroundColor = separatorColor
-            view.addSubview(separator)
-            separator.translatesAutoresizingMaskIntoConstraints = false
+        divider = MSFDivider()
+        if let divider = divider {
```

Comment:
Why is `divider` an optional to begin with, when we clearly create it unconditionally here? Should it be a `lazy var` instead?
```swift
private lazy var divider: MSFDivider = .init()
```